backend/dataall/modules/s3_datasets/db/dataset_repositories.py

```diff
@@ -190,22 +190,20 @@ def get_dataset_tables(session, dataset_uri):
         return session.query(DatasetTable).filter(DatasetTable.datasetUri == dataset_uri).all()
 
     @staticmethod
-    def update_dataset_shares_expiration(session, dataset):
+    def update_dataset_shares_expiration(session, enabledExpiration , datasetUri, expirationDate):
```

Comment:
In this function we change only ShareObjects. So, I should be a part of ShareObjectRepository

Reply:
Okay. Let me move out of dataset repositories and move it to ShareObjectRepository.